tools/applicationinsights-web-snippet/src/type.ts

```diff
@@ -16,7 +16,7 @@ export interface ISnippetConfig {
     cfg: IConfiguration;
     cr?: boolean; // cdn retry would be proceed if ture
     dle?: boolean; // Custom optional value to disable sdk load error to be sent
-    type?: string; // Custom optional value to specify the type of the snippet, e.g. "cjs.js"
+    integrity?: boolean; // Custom optional value to specify whether add the integrity attribute to the script tag
```

Comment:
Lets keep the size smaller perhaps `intg` or `fi` (fetch integrity)or `ui` (user integrity) or

maybe better use the security name
- `sri` for (sub resource integrity)  https://www.w3schools.com/Tags/att_script_integrity.asp

Reply:
We should also document that when you use this the load order and script execution is affected, which means the usage of `ld` will be ignored (or have no affect) and if the page "navigates" away before we can load the integrity file etc that they may lose the events.